decidim-elections/app/forms/decidim/votings/admin/voting_form.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    module Admin
+      # This class holds a Form to create/update votings from Decidim's admin panel.
+      class VotingForm < Decidim::Form
+        include TranslatableAttributes
+
+        translatable_attribute :title, String
+        translatable_attribute :description, String
+        attribute :slug, String
+        attribute :start_time, Decidim::Attributes::TimeWithZone
+        attribute :end_time, Decidim::Attributes::TimeWithZone
+        attribute :scope_id, Integer
+        attribute :banner_image, String
+        attribute :introductory_image, String
+        # attribute :attachment, AttachmentForm
+        # attribute :photos, Array[String]
+        # attribute :add_photos, Array
+
+        validates :title, translatable_presence: true
+        validates :description, translatable_presence: true
+        validates :slug, presence: true, format: { with: Decidim::Votings::Voting.slug_format }
+        validate :slug_uniqueness
+        validates :start_time, presence: true, date: { before: :end_time }
+        validates :end_time, presence: true, date: { after: :start_time }
+        validates :banner_image, passthru: { to: Decidim::Votings::Voting }
+        validates :introductory_image, passthru: { to: Decidim::Votings::Voting }
+        # validate :notify_missing_attachment_if_errored
```

Comment:
This method is defined but it's not being applied, not sure if you'll handle this in another PR @agustibr @orlera

Reply:
Thank you @mrcasals! Yes, it is already handled in an upcoming PR 👍